ocaml/xapi/storage_migrate.ml

```diff
@@ -542,6 +551,10 @@ let start' ~task ~dbg ~sr ~vdi ~dp ~url ~dest =
     error " Caught exception %s:%s. Performing cleanup." Api_errors.sr_not_attached sr_uuid;
     perform_cleanup_actions !on_fail;
     raise (Api_errors.Server_error(Api_errors.sr_not_attached,[sr_uuid]))
+  | Api_errors.Server_error(code, param) when code = Api_errors.sr_source_space_insufficient ->
```

Comment:
Why do we have a leading space in the error strings of this pattern match and the above?